dev/cli.sh

```diff
@@ -2,5 +2,5 @@
 
 git_id=$(git rev-parse --short=7 HEAD)
 
-image=docker.montagu.dide.ic.ac.uk:5000/orderly-web-cli:$git_id
+image=docker.montagu.dide.ic.ac.uk:5000/orderly-web-cli:fixcli
```

Comment:
This cli is broken at the moment. Once this is merged: https://github.com/vimc/orderly-web/pull/41 can return this to $git_id